src/nft/components/collection/CollectionAsset.tsx

```diff
@@ -71,6 +87,21 @@ export const CollectionAsset = ({
     }
   }, [addAssetsToBag, asset, bagExpanded, bagManuallyClosed, isMobile, setBagExpanded, trace])
 
+  useEffect(() => {
+    if (isSelected !== prevIsSelected && !inSweep) {
```

Comment:
The sweep condition doesn't work exactly as when I manually add items while the modal is open I don't get the memo. In addition, after I've closed the sweep modal, the tooltips don't show.

Reply:
u r right it was inconsistent. my changes now should have it working consistently.